src/Host.cpp

```diff
@@ -1692,44 +1692,44 @@ void Host::setUserDictionaryOptions(const bool _useDictionary, const bool useSha
 {
     Q_UNUSED(_useDictionary);
     bool useDictionary = true;
-    QMutexLocker locker(& mLock);
-    bool isChanged = false;
+    QMutexLocker locker(&mLock);
+    bool dictionaryChanged {};
```

Comment:
Never seen this syntax before

Reply:
We've used it in a few places in Mudlet now. It creates `dictionaryChanged` and sets it to a default value you'd expect. If you just do `bool dictionaryChanged;` the value could be random because one wasn't given (it basically doesn't do anything _at all_ to set one).

Technical term is "braced intializer", unfortunately all resources I could find from a quick search weren't an easy read.

Reply:
It uses the *default constructor* IIRC - in the same manner as it would be for a member in a class's constructor's initialization list, e.g. if `dictionaryChanged`  was a member then this would do the same as:
```
, dictionaryChanged()
```
would do in the initialization list.

Arguments can also be provided for the constructor of more complex items - and even `bool`s can be initialized to the *non-default* value `true` with:
```
bool dictionaryChanged {true};
```

:neckbeard: